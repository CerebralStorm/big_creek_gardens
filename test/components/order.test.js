import React from 'react'
import Order from '../../app/javascript/components/order'

let match = {
  params: {
    orderId: 1
  }
}

test('render a Order', () => {
  const wrapper = shallow(<Order match={match} />);
  expect(wrapper).toMatchSnapshot();
});