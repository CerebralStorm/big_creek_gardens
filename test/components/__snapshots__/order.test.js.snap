// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a Order 1`] = `
<div>
  <div
    className="container mt-3 mb-3 confirmation-container"
  >
    <div
      className="row marketing"
    >
      <div
        className="col-12"
      >
        <center>
          <h4>
            Order Placed: 
            <t
              ago={false}
              calendar={false}
              decimal={false}
              element="time"
              filter={[Function]}
              fromNow={false}
              interval={60000}
              onChange={[Function]}
              toNow={false}
              unit={null}
              unix={false}
              utc={false}
            />
          </h4>
          <h5>
            Order Number: #
          </h5>
          <hr />
        </center>
      </div>
      <div
        className="col-12"
      >
        <center>
          <div
            className="card"
          >
            <div
              className="card-body"
            >
              <address>
                <strong>
                  Shipping Address:
                </strong>
                <br />
                <br />
                <br />
                <br />
                <br />
                , 
                , 
              </address>
            </div>
          </div>
        </center>
      </div>
      <div
        className="col-12"
      >
        <table
          className="table"
        >
          <thead
            className="thead-dark"
          >
            <tr>
              <th
                scope="col"
              >
                Product Name
              </th>
              <th
                scope="col"
              >
                Quantity
              </th>
              <th
                scope="col"
              >
                Price
              </th>
              <th
                scope="col"
              >
                Total
              </th>
            </tr>
          </thead>
          <tbody />
        </table>
        <hr />
        <strong
          className="pull-right"
        >
          Total: $
        </strong>
      </div>
    </div>
  </div>
</div>
`;
